ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/dhcp-servers.yml

```diff
@@ -32,6 +32,20 @@ dhcp_servers:
         default_gateway: 10.0.0.1
 
   - vrf: default
```

Comment:
Can I ask you to add NTP to the example, since this will quickly become "inspiration", and NTP is a requirement for CV these days.

Reply:
No problem.  Just an fyi: I don't believe we support NTP via friendly CLI commands like we do for dns or tftp.  I believe we would need to use vendor options as seen below which is already supported in eos_cli_config_gen.

```
dhcp server
   vendor-option ipv4 NTP
      sub-option 42 type ipv4-address data 1.2.3.4
!
```

I can add the above example as a test case though.